core/src/main/res/values/strings.xml

```diff
@@ -314,10 +314,25 @@
     <string name="enable_sonic">Enable Sonic</string>
 
     <!-- Empty list labels -->
-    <string name="no_items_label">There are no items in this list.</string>
+    <string name="no_items_header_label">No queued episodes</string>
+    <string name="no_items_label">You can add episodes to the queue by long-pressing or downloading them.</string>
     <string name="no_feeds_label">You haven\'t subscribed to any podcasts yet.</string>
     <string name="no_chapters_label">This episode has no chapters.</string>
     <string name="no_shownotes_label">This episode has no shownotes.</string>
+    <string name="no_run_downloads_head_label">No download running</string>
+    <string name="no_run_downloads_label">You can download episodes on the podcast details screen.</string>
+    <string name="no_comp_downloads_head_label">No download episodes</string>
+    <string name="no_comp_downloads_label">You can download episodes on the podcast details screen.</string>
+    <string name="no_log_downloads_head_label">No download log</string>
+    <string name="no_log_downloads_label">Download logs will appear here when available.</string>
+    <string name="no_history_head_label">No History</string>
+    <string name="no_history_label">After you listen to an episode, it will appear here.</string>
+    <string name="no_all_episodes_head_label">No Episodes</string>
+    <string name="no_all_episodes_label">When you add a podcast, the episodes will be shown here.</string>
+    <string name="no_new_episodes_head_label">No new episodes</string>
+    <string name="no_new_episodes_label">When new episodes arrive, they will be shown here.</string>
+    <string name="no_fav_episodes_head_label">No favourite episodes</string>
+    <string name="no_fav_episodes_label">You can add episodes to the favourites by long-pressing them.</string>
```

Comment:
We are using american english throughout the rest of the app, so this should be `favorites` instead of `favourites`.